### ISSUE TITLE
Help texts for the Observations form

### ISSUE THREAD

**Issue by coffeejunk** [sharkipedia/sharkipedia#60]
One open topic is how to make clear what kind of input is needed in a field, if no drop-down is available. One thing we can easily do is add short help texts below the input boxes like this:

<img width="686" alt="Screenshot 2019-10-09 17 07 02" src="https://user-images.githubusercontent.com/205556/66520475-39bb4600-eab7-11e9-9b3d-8c9788d15681.png">

All we have to do is add a tiny bit of HTML. 

The observation form is made of two relevant components:

1. [`app/views/observations/_form.html.erb`](https://github.com/sharktraits/sharkT/blob/0d30cb0bdf8f0fcc1d13f5ce43d0e9d31efdadbf/app/views/observations/_form.html.erb) for all the fields related to an observation
1. [`app/views/observations/_measurement_fields.html.erb`](https://github.com/sharktraits/sharkT/blob/master/app/views/observations/_measurement_fields.html.erb) for all fields related to the measurements which get attached to an observation

All we have to do is add a `<p class="help">` as the last element of the `<div class="field">` which encloses the input in question.

```html
<div class="field">
  <%= f.label :depth, class: 'label' %>
  <div class="control">
    <%= f.text_field :depth, class: 'input' %><br />
  </div>
</div>
```
=>
```html
<div class="field">
  <%= f.label :depth, class: 'label' %>
  <div class="control">
    <%= f.text_field :depth, class: 'input' %><br />
  </div>
  <p class="help">Depth of the observation in <code>m</code>.</p>
</div>
```

Since the help is only HTML we can easily add additional styling like for example wrapping the unit (`m`) in a `<code>` element for additional highlighting.

@creeas let me know if you need any help with this.

**Comment by creeas** [comment]
Quick question @coffeejunk . On the measurement fields form, am I correct that the ```<p class="help">``` goes after the  ```<span>``` tags? These seem to control the icon within the dropdown menu indicating a required field. So for example for trait-class I would have:

```
<div class="field">
        <%= f.label :trait_class, class: 'label' %>
        <p class="control has-icons-left">
          <%= f.collection_select(:trait_class_id, @trait_classes, :id, :name, {:prompt => 'Select Trait Class'}, style: 'width: 100%', class: 'independent') %>
          <span class="icon is-small is-left">
            <i class="fas fa-exclamation"></i>
          </span>
          <p class="help"> The class of trait for the measurement (Length, Age, Growth, Reproduction, Demography, or Relationships). </p>
        </p>
      </div>
```
Thanks!

**Comment by coffeejunk** [comment]
@creeas you want to be inside of the `<div>` with the `class="field"` but outside of any elements inside that div, or said differently the help tag should be the last element of the field-div:

```
      <div class="field">
        <%= f.label :trait_class, class: 'label' %>
        <p class="control has-icons-left">
          <%= f.collection_select(:trait_class_id, @trait_classes, :id, :name, {:prompt => 'Select Trait Class'}, style: 'width: 100%', class: 'independent') %>
          <span class="icon is-small is-left">
            <i class="fas fa-exclamation"></i>
          </span>
        </p>
        <p class="help"> The class of trait for the measurement (Length, Age, Growth, Reproduction, Demography, or Relationships). </p>
      </div>
```

**Comment by coffeejunk** [comment]
Closing; implemented as of #80
